qml/YubiKey.qml

```diff
@@ -309,6 +309,11 @@ Python {
                     currentDevice.hasPassword = true
                     currentDeviceValidated = false
                     navigator.goToEnterPasswordIfNotInSettings()
+                } else if (resp.error_id === 'open_device_failed') {
+                    clearCurrentDeviceAndEntries()
+                    yubiKey.availableDevices = []
+                    navigator.snackBarError("Failed connecting to YubiKey.")
```

Comment:
How about this instead? We have this format in a few places. I like it. open_device_failed will print ` Failed to connect to YubiKey`
```suggestion
                    navigator.snackBarError(navigator.getErrorMessage(
+                                            resp.error_id))
```

Reply:
Good idea, changed